### ISSUE TITLE
Decals dr10

### ISSUE THREAD

**Issue by healytwin1** [kmhess/SoFiA-image-pipeline#76]
Decals dr10 is now a reality on the legacy surveys sky viewer (formal data release is still pending but I suspect not far away). DR10 covers the whole sky except for the galactic plane which is covered by DECAPS2 (http://decaps.skymaps.info/viewer.html). May be nice to switch the default colour image to decals. There is a caveat in that the max size for a decals image is 3000 pixels which at 0.25arcsec is 12.5arcmin, so if an image needs to be larger than that, we may need to consider a variable pixel size.

**Comment by kmhess** [comment]
Hi @healytwin1,  Nice!  Thanks for the heads up.  DECAPS looks cool too.  Do you have a reference/website link for the 3000 pixel limit?

**Comment by healytwin1** [comment]
I don't, only experience...

**Comment by paoloserra** [comment]
Concerning image and pixel size, these options are already https://github.com/kmhess/SoFiA-image-pipeline/blob/34178541354860759f91798c8376e17a9a53c4b8/src/modules/get_ancillary.py#L99-L100

In this line we are setting the pixel size to default, but it could be changed to cover a wider sky area with a fixed nr of pixels (I've checked that this works from the browser).

I've also tried to get images larger than 3000 pix from the browser. There's no error message, an image is downloaded, but it is 3000 pix.

I find the same using the JPEG URL given at https://www.legacysurvey.org/dr10/description/#toc-entry-2 (and adding the image size options, trying to push beyond 3000 pix.)

That same webpage states that the maximum size of the cutouts is 512 pix, but it really is 3000.

The 3000 pix limit is stated in, e.g., the DR6 description webpage https://www.legacysurvey.org/dr6/description/#toc-entry-2
